app/assets/stylesheets/govuk_publishing_components/components/print/_step-by-step-nav.scss

```diff
@@ -1,116 +1 @@
-$grey-2: #bfc1c3;
-$white: #ffffff;
-$number-circle-size: 35px;
-$stroke-width: 1px;
-
-.gem-c-step-nav:not(.gem-c-step-nav--large),
-.gem-c-step-nav__controls,
-.gem-c-step-nav__toggle-link,
-.gem-c-step-nav__help {
-  display: none;
-}
-
-.gem-c-step-nav {
-  position: relative;
-
-  &:before {
-    content: "";
-    position: absolute;
-    top: 0;
-    left: $number-circle-size / 2;
-    width: 3px;
-    height: 100%;
-    margin-left: -1.5px;
-    border-left: solid $stroke-width $grey-2;
-  }
-
-  &:after {
-    content: "";
-    position: absolute;
-    z-index: 6;
-    bottom: 0;
-    left: 0;
-    margin-left: $number-circle-size / 4;
-    width: $number-circle-size / 2;
-    height: 0;
-    border-bottom: solid $stroke-width $grey-2;
-  }
-}
-
-.gem-c-step-nav__step {
-  padding-left: 60px;
-  position: relative;
-}
-
-.gem-c-step-nav__steps {
-  padding: 0;
-  list-style: none;
-}
-
-.gem-c-step-nav__circle {
-  box-sizing: border-box;
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: $number-circle-size;
-  height: $number-circle-size;
-  background: govuk-colour("white");
-  border-radius: 100px;
-  text-align: center;
-  font-size: 19px;
-  font-weight: bold;
-  line-height: 1.5;
-}
-
-.gem-c-step-nav__circle--number {
-  border: solid 1px $grey-2;
-}
-
-.gem-c-step-nav__step,
-.gem-c-step-nav__paragraph,
-.gem-c-step-nav__links {
-  @include govuk-font(16);
-  padding-bottom: 1em;
-}
-
-.gem-c-step-nav__title {
-  @include govuk-font(19, $weight: bold);
-  margin: 0 0 .5em;
-  padding: 0;
-}
-
-.gem-c-step-nav__button--title {
-  @include govuk-font(19, $weight: bold);
-  padding: 0;
-  border: 0;
-  background: none;
-}
-
-.gem-c-step-nav__context {
-  &:before {
-    content: " \2013  ";
-  }
-}
-
-.gem-c-step-nav__panel-link--active {
-  font-weight: bold;
-}
-
-.gem-c-step-nav__links {
-  padding-left: 0;
-  list-style: none;
-}
-
-.gem-c-step-nav__links--choice {
-  padding-left: 30px;
-  list-style: disc;
-}
-
-.gem-c-step-nav__link {
-  margin-bottom: .3em;
-}
-
-.gem-c-step-nav__circle-step-label,
-.gem-c-step-nav__circle-step-colon {
-  display: none;
-}
```

Comment:
Just out of curiosity - what's the effect of removing these styles and not copying them over into the `print` mixin? The PR does mention that some styles were being duplicated and can be removed entirely; is that what's happening here?

Reply:
Overall the effect is positive, we ended up with fewer lines off CSS, while improving the print styles.

Further details are included on this closed PR's description, including screenshots showing before and after:

https://github.com/alphagov/govuk_publishing_components/pull/3118#issue-1479197815

Reply:
Great, thanks Martin 👍